test/test-loaders-node.js

```diff
@@ -282,7 +282,7 @@ describe('path-loader (node.js loaders)', function () {
     describe('https', function () {
       it('make sure we get a loader', function (done) {
         pathLoader
-          .load('https://cdn.rawgit.com/whitlockjc/path-loader/master/package.json')
```

Comment:
Somehow, CI on Node 8 only (not on 10+) throws a "Certificate expired" error:

```
Error: certificate has expired
      at TLSSocket.<anonymous> (_tls_wrap.js:1116:38)
      at TLSSocket._finishInit (_tls_wrap.js:643:8)
      at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:473:38)
```

I don't think this is `path-loader`'s problem, and the other url works.

Reply:
Agreed, but we should try to figure out how to workaround this because we'd like the tests to pass on Node 8, since it's still in TLS.

Reply:
I think this is a problem with certificates on Travis rather than with Node 8. It works with `https://rawgit.com`, which uses a Cloudflare certificate.